test/fuzz/classifier-spdx.c

```diff
@@ -0,0 +1,76 @@
+#include "../licences.h"
```

Comment:
Please add the licensing header/comment here. You can also update the year next to your name in the `Makefile`.

Reply:
Done